src/models/user.ts

```diff
@@ -0,0 +1,16 @@
+const user = (sequelize, Sequelize) => {
+  return sequelize.define('user', {
+    id: {
+      type: Sequelize.UUID,
+      defaultValue: Sequelize.UUIDV4,
+      primaryKey: true,
+    },
+    username: Sequelize.STRING(255),
+    displayname: Sequelize.STRING(255),
+    email: Sequelize.STRING(255),
+    timestamps: true,
+    createdAt: 'date_created',
+    updatedAt: 'date_updated',
+  });
+}
+export = user;
```

Comment:
Should use proper syntax. See [TypeScript Deep Dive](https://basarat.gitbooks.io/typescript/content/docs/project/external-modules.html) for details.